src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -973,6 +973,7 @@ TEST_F(ApplicationManagerImplTest,
        OnDeviceSwitchingFinish_ExpectUnregisterAppsInWaitList) {
   std::shared_ptr<MockApplication> switching_app_ptr =
       std::make_shared<MockApplication>();
+  app_manager_impl_->SetMockPolicyHandler(mock_policy_handler_);
```

Comment:
I don't think that there are test cases that involve deleting one `policy_handler` and setting another, hence i would suggest moving `SetMockPolicyHandler` to `SetUp`

Reply:
@mked-luxoft Done in 847ef5be3d621cabf18d1f8c46810c8443a428cf